### PR TITLE
Use Separate Vertex Formant Attribute in GL45backend

### PR DESCRIPTION
**Pull request by samcake**
work in porgress

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: a31c76f4e19247aa5fe50663025928feeece8974 available at: https://gist.github.com/9b99208ef966c74ce226e93edd45f570<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8295-a31c76f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8295-a31c76f.dmg

File: libraries/gpu-gl/src/gpu/gl/GLBackendInput.cpp [hunk 1]
```diff
@@ -170,6 +104,49 @@ void GLBackend::updateInput() {
...
+                    }
+
+                    if (i == 0) {
+                        frequency = attrib._frequency;
+                    } else {
+                        assert(frequency == attrib._frequency);
+                    }
+
+                    (void)CHECK_GL_ERROR();
+=======
```
Comment (ZappoMan) [inline libraries/gpu-gl/src/gpu/gl/GLBackendInput.cpp]: looks like a bad merge

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3a7c33165b70493e0b22c5437d7b01881b2d5594 available at: https://gist.github.com/9b99208ef966c74ce226e93edd45f570<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8295-3a7c331.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8295-3a7c331.dmg

**Comment by ZappoMan** [comment]
Closing for now... we can reopen after steam launch and rebasing.

**Pull request closed without merging**